lib/helpers.js

```diff
@@ -0,0 +1,23 @@
+'use babel';
+/* @flow */
+
+
+export function check(pathToFlow: string, args: Array<string>, options: Object): Promise<string> {
+  const helpers = require('atom-linter');
```

Comment:
I tried, that but it seemed to cause problems earlier. Now the tests are working, let me try again.